search-service/src/main/kotlin/com/egm/stellio/search/temporal/web/TemporalEntityOperationsHandler.kt

```diff
@@ -38,6 +40,13 @@ class TemporalEntityOperationsHandler(
     suspend fun queryEntitiesViaPost(
         @RequestHeader httpHeaders: HttpHeaders,
         @RequestBody requestBody: Mono<String>,
+        @AllowedParameters( // todo double check GEOMETRY_PROPERTY, OPTIONS  in spec
```

Comment:
most of them are passed in the body (see GeoQuery - 5.2.13 and TemporalQuery - 5.2.21)